results.html

```diff
@@ -2,42 +2,201 @@
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+</head>
+
+<body class="hero-image"></body>>
+
+<main>
+  <div class="rowOne">
+    <div class="row">
+      <div class="col s5 m5 l5">
+        <img class="logo" src="./assets/images/corporate-logo-maker-with-icon-1517d (10).png" />
+      </div>
+      <div class="col s7">
+        <h1 id="cityName">City Name</h1>
+      </div>
+    </div>
+  </div>
+
+  <div class="rowTwo">
+    <div class="row"></div>
+    <div class="col">
+      <ul class="collapsible">
+        <li>
+          <div class="collapsible-header white-text opacityLi"><i class="large material-icons">search</i><input
+              type="text" class="search-bar" /></div>
+          <div class="collapsible-body">
+            <div class="search-container">
+
+              <div class="search-results container">
+                <div class="container-item">item1</div>
+                <div class="container-item">item2</div>
+                <div class="container-item">item3</div>
+              </div>
+            </div>
+          </div>
+        </li>
+      </ul>
+    </div>
+    <div class="col">
+      <ul class="collapsible">
+        <li>
+          <div class="collapsible-header white-text opacityLi"><i class="large material-icons">view_list</i>City
+            Information</div>
+          <div class="collapsible-body">
+            <div id="CityData" class="card">
+              <p>Demographic Data: 1</p>
+              <p>Demographic Data: 2</p>
+              <p>Demographic Data: 3</p>
+              <p>Demographic Data: 4</p>
+              <p>Demographic Data: 5</p>
+            </div>
+          </div>
+        </li>
+      </ul>
+    </div>
+    <div class="col">
+      <ul class="collapsible">
+        <li class>
+          <div class="collapsible-header white-text opacityLi"><i class="large material-icons">map</i>Google Map</div>
+          <div class="collapsible-body">
+            <div class="card">
+              <iframe
+                src="https://www.google.com/maps/embed?pb=!1m18!1m12!1m3!1d239283.1392350707!2d-87.01468175940212!3d20.432348997106995!2m3!1f0!2f0!3f0!3m2!1i1024!2i768!4f13.1!3m3!1m2!1s0x8f4ef8efb0d19327%3A0x342ebcae5c018bfc!2sCozumel!5e0!3m2!1sen!2sus!4v1657560021697!5m2!1sen!2sus"
+                width="100%" height="100%" style="border: 0" allowfullscreen="" loading="lazy"
+                referrerpolicy="no-referrer-when-downgrade"></iframe>
+            </div>
+
+          </div>
+    </div>
+  </div>
+  <div class="rowThree">
+    <div class="row">
+      <div class="col s12 m4 l4">
+        <div class="carousel-info">
+          <div id="Restaurants" class="carousel carousel-slider center">
+            <div class="carousel-item opacity white-text" href="#one!">
+              <h2>Restaurant 1</h2>
+              <p class="white-text">Info 1</p>
+              <img src="/assets/images/restaurant/3-course.jpeg" alt="" class="class">
+            </div>
```

Comment:
`class="class"`?   

You can leave `<div class="carousel-info"></div>` empty.  The content is rendered dynamically.  **Anything you put inside this element will be removed** when I call `$('.carousel-info').html('')`

If you want to put `<img>`s in the carousels I would suggest adding them in CSS as `background-image: url(xxx.jpeg);` or I can add them dynamically